### ISSUE TITLE
"MisconfigurationException" error

### ISSUE THREAD

**Issue by fncode246** [asteroid-team/asteroid#408]

**Comment by fncode246** [comment]
I want to train ConvTasNet with librimix, I made librimix and part of it is in google drive now.
To train with train.py , the error "" has occured. How can i solve it?
thanks

**Comment by fncode246** [comment]
I want to train ConvTasNet with librimix, I made librimix and part of it is in google drive now.
To train with train.py , the error "**MisconfigurationException: `configure_optimizers` must include a monitor when a `ReduceLROnPlateau` scheduler is used. For example: {"optimizer": optimizer, "lr_scheduler": scheduler, "monitor": "metric_to_track"} site:stackoverflow.com**" has occured. How can i solve it?
thanks

**Comment by mpariente** [comment]
Versions of asteroid, pytorch lightning, exact command?

**Comment by jonashaag** [comment]
Can you please include the info requested here? https://github.com/asteroid-team/asteroid/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

(We should include that stuff in the question template as well.)

**Comment by mpariente** [comment]
Closing, because there is no info.
